bundles/org.openhab.voice.porcupineks/README.md

```diff
@@ -0,0 +1,25 @@
+# Porcupine Keyword Spotter
+
+This voice service allows you to use the PicoVoice product Porcupine as your keyword spotter in openHAB.
+
+## How to use it
+
+After installing you would be able to access the addon options through the openHAB configuration page under the 'Other Services' section. There you will need to provide your PicoVoice Api Key.
```

Comment:
General comment for all the README: add a return after sentence, it helps for maintenance.

After installing you **will** be ...